### PR TITLE
Add answers to Stack and Queue Data Structures

### PR DESCRIPTION
**Pull request by rahulkarda**
Added explanation, uses, example, and sources to Stack and Queue Data Structures question.

### CODE REVIEW

File: BANKI-answers.md [hunk 1]
```diff
@@ -1401,16 +1401,16 @@ function factorial(num) {
...
-  - **Source:**
+- [x] What is a Queue?
+  - **Explanation:** A Queue is a linear structure which follows a particular order in which the operations are performed. The order is First In First Out (FIFO).
+  - **Use:** Queue is used when things don't have to be processed immediately, but have to be processed in First In First Out order like Breadth First Search.
+  - **Example:** The Event Loop Model prioritizes all the Jobs in Job Queue. 
+  - **Source:** https://www.geeksforgeeks.org/queue-data-structure/
+- [x] What is a Stack?
+  - **Explanation:** Stack is a linear data structure which follows a particular order in which the operations are performed. The order is LIFO(Last In First Out).
+  - **Use:** Stacks are used to implement functions, parsers, expression evaluation, and backtracking algorithms
+  - **Example:** The Even loop uses call stack. Every time a script or function calls a function, it's added to the top of the call stack. Every time the function 	exits, the interpreter removes it from the call stack.
```
Comment (curtisbarnard) [inline BANKI-answers.md]: typo here for "event"

**Review by curtisbarnard — CHANGES_REQUESTED** [review]

File: BANKI-answers.md (hunk 1 above)
Comment (rahulkarda) [reply BANKI-answers.md]: Oh thanks for catching that, I'll fix it asap.

**Review by rahulkarda — COMMENTED** [review]
Checked for typos and broken links. Everything is good.

**Comment by curtisbarnard** [comment]
Thank you!

**Comment by rahulkarda** [comment]
Awesome!

**Pull request merged**